docs/book/usage/setup.md

```diff
@@ -194,7 +194,7 @@ CLI configuration refers to the standard location for Sui configuration
 The easiest way to obtain the latest configuration is by downloading it directly from Walrus:
 
```

Comment:
Suggestion: Can we add something like the following after the first paragraph of this section?
```md
If you installed Walrus through the [install script](#nix-install), this configuration file was already created. If not, ...
```